### ISSUE TITLE
Enable continuing analysis in multiapplication projects even is some license files are unavailable 

### ISSUE THREAD

**Issue by Richter-cg** [devonfw/solicitor#56]
It could happen that some input files are missing during the analysis of multiple projects at once.
Therefor it would be great to complete the analysis with the available files and show a warning on the title sheet of the output files.

**Comment by ohecker** [comment]
Feature can be activated via a new property (which can be set e.g. via command line - see user guide), Extract from the application properties;

```
# If there is an exception while reading the raw inventory data for an application the processing will be aborted by default.
# By setting this property to true processing will continue in case that the input file is not found. This allows processing
# in multi application projects where some input files are not yet available. A warning (SOLI-045) will be logged in this case and
# Fields releaseId, releaseDate, sourceRepo, programmingEcosystem of the Application object will be filled with string
# "(INPUT DATA MISSING!)" and the name will be suffixed with "(INPUT DATA MISSING!) in this case"
# If using this feature make sure to double check the log files / result to assure data is complete.
solicitor.tolerate-missing-input=false 

```

**Comment by ohecker** [comment]
Included in 1.0.7
